Scripts/build.sh

```diff
@@ -87,7 +87,7 @@ function xcbuild() {
 
 function fastlane_test() {
   if [[ -n "$XC_DESTINATION" ]]; then
-    SDK="$XC_SDK" DEST="$XC_DESTINATION" SCHEME="$1" bundle exec fastlane test
+    SDK="$XC_SDK" DEST="$XC_DESTINATION" SCHEME="$1" fastlane test
   else
     SDK="$XC_SDK" SCHEME="$1" bundle exec fastlane test
```

Comment:
should we also remove bundle exec from here?

Reply:
I noticed Azure instance already has fastalne, so we can use pre-installed one in our usage. Let me try it.